core/vdbe/mod.rs

```diff
@@ -2925,6 +2937,45 @@ mod tests {
         assert_eq!(exec_sign(&input), expected);
     }
 
+    #[test]
+    fn test_exec_zeroblob() {
+        let input = OwnedValue::Integer(0);
+        let expected = OwnedValue::Blob(Rc::new(vec![]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Null;
+        let expected = OwnedValue::Blob(Rc::new(vec![]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Integer(4);
+        let expected = OwnedValue::Blob(Rc::new(vec![0; 4]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Integer(-1);
+        let expected = OwnedValue::Blob(Rc::new(vec![]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Text(Rc::new("5".to_string()));
+        let expected = OwnedValue::Blob(Rc::new(vec![0; 5]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Text(Rc::new("-5".to_string()));
+        let expected = OwnedValue::Blob(Rc::new(vec![]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Text(Rc::new("text".to_string()));
+        let expected = OwnedValue::Blob(Rc::new(vec![]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Float(2.6);
+        let expected = OwnedValue::Blob(Rc::new(vec![0; 2]));
+        assert_eq!(exec_zeroblob(&input), expected);
+
+        let input = OwnedValue::Blob(Rc::new(vec![1]));
+        let expected = OwnedValue::Blob(Rc::new(vec![]));
+        assert_eq!(exec_zeroblob(&input), expected);
+    }
```

Comment:
Ah, found out that `select zeroblob(x'32');` return `x'0000'` because `x'32'` is ASCII `2`. I will add this case.